lib/main_common.pm

```diff
@@ -1100,10 +1100,11 @@ sub load_inst_tests {
     if (installyaststep_is_applicable()) {
         loadtest "installation/resolve_dependency_issues" unless (get_var("DEPENDENCY_RESOLVER_FLAG") || get_var('KERNEL_64KB_PAGE_SIZE'));
         loadtest "installation/installation_overview";
+        loadtest 'installation/configure_bls' if (is_bootloader_sdboot || is_bootloader_grub2_bls);
```

Comment:
This uses `KEEP_GRUB_TIMEOUT` so should be after the `set_var` below (even though it has no effect in practice)